common/collection_exercise_utilities.py

```diff
@@ -100,7 +101,7 @@ def generate_collection_exercise_dates_from_period(period):
     """Generates a collection exercise events base date from the period supplied."""
 
     now = datetime.now()
-    now = now.replace(tzinfo=tz.gettz('Europe/London')).astimezone(tz.gettz('UTC'))
+    now = now.replace(tzinfo=tzlocal()).astimezone(tz.gettz('UTC'))
```

Comment:
Will this cause problems based on which server location we host at?

Reply:
For the time being this way should work, gonna put a card on the board to look at the way we do timezone conversion in the acceptance tests